iop_parse.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
```

Comment:
I'd like to keep things like this in the examples directory, both iop_parse.py and oup_parse.py -- I think iop_parse.py already exists as examples/ex_iop.py

Reply:
These are what I'm using to create the files to load into classic. I can put them into the examples directory, but they include some file handling I think is needed (e.g. to load by volume/issue), and the tmp handling for oup.

Reply:
How does it find pyingest.parsers.iop if I run it from the adsabs-pyingest directory?  OTher scripts in there also failing to find pyingest.parsers.xxx